### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by zmstone**

### CODE REVIEW

File: include/emqx_ct_job.hrl [hunk 1]
```diff
@@ -14,4 +14,9 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
+-ifndef(EMQX_CT_JOB_HRL).
```
Comment (jovdipp) [inline include/emqx_ct_job.hrl:17]: Why the protection of the ifndef ?

File: include/emqx_ct_jobs_transform.hrl [hunk 2]
```diff
@@ -14,6 +14,9 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
+-ifndef(EMQX_CT_JOBS_TRANSFORM).
+-define(EMQX_CT_JOBS_TRANSFORM, true).
```
Comment (jovdipp) [inline include/emqx_ct_jobs_transform.hrl:18]: Whats the intent with the EMQX_CT_JOBS_TRANSFORM  ?
A test suite using jobs matrix, adds -compile([{parse_transform, emqx_ct_jobs_suite_transform}]).
and thus no need for a flag ?

File: src/emqx_ct_helpers_jobs.erl [hunk 3]
```diff
@@ -20,8 +20,8 @@
          is_job_node/0, node_name/1, ct_master_orchestration/3, repo_root/0, container_ip/1,
          container_ip/2, test_platform_host/0]).
 
--include("./emqx_ct_job.hrl").
--include("./emqx_ct_jobs_transform.hrl").
```
Comment (jovdipp) [inline src/emqx_ct_helpers_jobs.erl:24]: I’m not even sure why I had “./” in the includes, might have been the IDE refactor ( rename ). Thanks for cleaning it

File: include/emqx_ct_job.hrl (hunk 1 above)
Comment (zmstone) [reply include/emqx_ct_job.hrl:17]: It’s to avoid error if included twice or more

File: include/emqx_ct_jobs_transform.hrl (hunk 2 above)
Comment (zmstone) [reply include/emqx_ct_jobs_transform.hrl:18]: Should have been `EMQX_CT_JOBS_TRANSFORM_HRL`

File: src/emqx_ct_helpers_docker.erl [hunk 4]
```diff
@@ -31,24 +31,65 @@ inspect( ContainerName, Inspection ) ->
...
 
 set_env_file(Env) ->
     { ok, CWD } = file:get_cwd(),
     Unique = erlang:unique_integer()*-1,
     Filename = lists:concat([ ".docker-env-", Unique ]),
     Path = filename:join([ CWD, Filename]),
-    file:write_file(Path, Env),
+    ok = file:write_file(Path, Env),
     { ok, Path }.
+
```
Comment (jovdipp) [inline src/emqx_ct_helpers_docker.erl:53]: have you considered this before, the last time I wanted to better exit status it used :

```erlang
os_cmd(Command) ->
	Port = open_port({spawn, Command}, [ use_stdio, stderr_to_stdout, eof, hide, exit_status, { parallelism, true } ]),
	get_data(Port, []).

get_data(Port, Sofar) ->
	receive
		{Port, {data, Bytes}} ->
			get_data(Port, [Sofar|Bytes]);
		{Port, eof} ->
			Port ! {self(), close},
			receive
				{ Port, closed } ->
					true
			end,
			receive
				{'EXIT',  Port,  _} ->
					ok
			after 1 ->              % force context switch
				ok
			end,
			ExitCode = receive
						 { Port, { exit_status, Code } } ->
							Code
					   end,
			{ ExitCode, lists:flatten( Sofar) }
	end.
```

Comment (zmstone) [reply src/emqx_ct_helpers_docker.erl:53]: Yes. I used both before.
but I find `echo $?` is easier to understand.

**Pull request merged**